Tests/Unit/Doctrine/Phpcr/Form/Type/StaticContentTypeTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2016 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\ContentBundle\Tests\Unit\Doctrine\Phpcr\Form\Type;
+
+use Symfony\Cmf\Bundle\ContentBundle\Doctrine\Phpcr\Form\Type\StaticContentType;
+use Symfony\Cmf\Bundle\ContentBundle\Doctrine\Phpcr\StaticContent;
+use Symfony\Component\Form\Test\TypeTestCase;
+
+class StaticContentTypeTest extends TypeTestCase
+{
+    public function testSubmitValidData()
+    {
+        $data = [
+            'name' => 'hello-world',
+            'parentDocument' => '/cms/content',
```

Comment:
I use a string here as parentDocument, but in docblock I see object. What are the requirements for an object to be used as parent? Is there an interface to implement or it's just the mapping that allows it?